src/DynamoRevitInstall/Product.wxs

```diff
@@ -115,23 +133,36 @@
           </Directory>
         </Directory>
         <Directory Id="DYNAMO_PROGDATA" Name="Dynamo">
-          <Directory Id="PROGDATA_07" Name="0.7"/>
+          <Directory Id="PROGDATA" Name="$(var.Major).$(var.Minor)"/>
         </Directory>
       </Directory>
     </Directory>
 
+    <DirectoryRef Id="TARGETDIR">
+      <Component Id="ProductRegistry" Win64="yes">
+        <RegistryKey Root="HKLM"
+                     Key="SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(var.ProductName) $(var.Major).$(var.Minor)"
+                     ForceDeleteOnUninstall="yes">
+          <RegistryValue Name="InstallLocation" Value="[DYNAMOREVIT_INSTALLDIR]" Type="string" />
+          <RegistryValue Name="UninstallString" Value="MsiExec.exe" Type="string" />
+          <RegistryValue Name="UninstallParam" Value="/X[ProductCode] /quiet" Type="string" />
+          <RegistryValue Name="Version" Value="$(var.FullVersion)" Type="string" />
+          <RegistryValue Name="RevVersion" Value="$(var.Rev)" Type="integer" />
+        </RegistryKey>
+      </Component>
+    </DirectoryRef>
 
-    <!-- Install the Addin files-->
     <ComponentGroup Id="REVIT_2014" Directory="ADDINS_2014">
-      <Component Id="DynamoForRevit2015.addin" Guid="976a5fd6-0a53-4530-b9c8-1004fc7a6200">
-        <File Id="DynamoForRevit2015.addin" Name="Dynamo.addin" Source="$(var.base)\tools\install\Extra\DynamoForRevit2014.addin"/>
+      <Component Id="DynamoForRevit2014.addin" Guid="976a5fd6-0a53-4530-b9c8-1004fc7a6200">
```

Comment:
Now that we are auto generating the addin files using AddinGenerator, we don't need to include these addins.